auth-react/docs/emailpassword/email-password-auth.md

```diff
@@ -32,4 +32,5 @@ __HIGHLIGHT__        return (
        userId: string,
        jwtPayload: any /*JSON object set on the backend*/
    }
-   ```
\ No newline at end of file
+   ```
+- `EmailPasswordAuth` will update the session, because it uses `SessionAuth` underneath. Read more about session updates in [SessionAuth guide](/docs/auth-react/docs/session/session-auth).
```

Comment:
You also want to mention that this takes an `onSessionExpired` prop?

Reply:
I can duplicate them from `SessionAuth`, yes. It definitely makes sense.